### ISSUE TITLE
Score Leaders

### ISSUE THREAD

**Issue by connermcd** [Trench-Wars/twcore#19]
## Trac Ticket [#753](http://twcore.org/ticket/753) assigned enhancement

> Reported by: Burnt
> Owned by:    fis
> Component:   Bots - PubSystem/PubAutobot
> Priority:    normal
> Severity:    trivial
## Description:

This is just an idea if anybody is interested :)

Categories: total points, wins, loses, kill points, flag points

players commands: 

!helpscores gets all player commands pertaining to the scores.

!leaders <category> gets leaders (top 5) for individual category.

!squadleaders <category> gets leaders for Squad category

!winners <category> gets winners from last reset

!squadwinners <category> gets squad winners from last reset

Squad stats should only include stats from the top 10 players from each squad in total points.

Basically just copy what the Chaos-Scores-Bot does in Chaos/League zone (type !helpresetwinners to see commands)

**Comment by connermcd** [comment]
_SpookedOne changed **priority** from_ high _to_ normal

**Comment by connermcd** [comment]
_SpookedOne changed **severity** from_ major _to_ trivial

**Comment by connermcd** [comment]
_SpookedOne changed **summary** from_ Program Bot in Pub to show score reset leaders/winners _to_ Pub score leaders

**Comment by connermcd** [comment]
_SpookedOne changed **type** from_ task _to_ enhancement

**Comment by connermcd** [comment]
_SpookedOne changed **summary** from_ Pub score leaders _to_ Pub Score

**Comment by connermcd** [comment]
_SpookedOne changed **owner** to_ fis

**Comment by connermcd** [comment]
_SpookedOne changed **status** from_ new _to_ assigned

**Comment by connermcd** [comment]
**fis>** Would be OK to have as an addition. No priority or need here. If this could maybe be done as a first project for a bot-dev starter, then so be it. Else I wouldn't spend dev time on this, rather focus on other stuff.

**Comment by connermcd** [comment]
_SpookedOne changed **summary** from_ Pub Score _to_ Score Leaders
